src/components/filesystem/index.js

```diff
@@ -850,7 +851,10 @@ module.exports = {
 		var that = this;
 		this.confirmView(file, () => {that.showGallery = true;});
 	    } else if (mimeType === "text/plain") {
-		this.showTextViewer = true;
+		if (file.getName().endsWith(".md"))
```

Comment:
what about other extensions?

Reply:
Yep. I will add some more. I mainly want plain text

Reply:
I made it open all text files.